packages/wp-source/src/libraries/handlers/author.ts

```diff
@@ -42,8 +41,7 @@ const authorHandler: Handler = async ({ route, params, state, libraries }) => {
   const totalPages = getTotalPages(response);
 
   // 5. add data to source
-  Object.assign(state.source.data[route], {
-    id,
+  Object.assign(state.source.data[route], state.source.data[path], {
```

Comment:
In case of requesting other than the first page, we are setting `isFetching` to false when it should be true.

Add a test to check that `isFetching` and `isReady` are never false until `fetch` finishes.

Reply:
Done!